src/runners/bato/parser.ts

```diff
@@ -255,14 +255,7 @@ export class Parser {
     const imgAccListString = AES.decrypt(batoWord, evaluatedPass).toString(
       enc.Utf8
     );
-
-    if (!imgAccListString) throw new Error("Invalid Acc List String");
-
     const imgAccList: string[] = JSON.parse(imgAccListString);
-
-    if (!Array.isArray(imgAccList) || imgAccList.length == 0)
-      throw new Error("Acc List Is Invalid");
-
     const urls = imgHttpList.map((v, i) => `${v}?${imgAccList[i]}`);
```

Comment:
Same goes here, don't you get a null ref exception or out of bounds exception if the collection doesn't exist and you try to access it?

Reply:
`imgAccList` still exists, I just don't care if it has any content to it.